spec/features/parent_gives_consent_spec.rb

```diff
@@ -0,0 +1,350 @@
+    choose "No"
+
+    click_on "Continue"
+  end
+
+  def and_i_fill_in_my_childs_date_of_birth_matching_the_sais_record
+    expect(page).to have_content("What is your child’s date of birth?")
+    fill_in "Day", with: @child.date_of_birth.day
+    fill_in "Month", with: @child.date_of_birth.month
+    fill_in "Year", with: @child.date_of_birth.year
+    click_on "Continue"
+  end
+
+  def and_i_confirm_they_attend_the_pilot_school
+    expect(page).to have_content("Confirm your child’s school")
+    expect(page).to have_content("Pilot School")
+
+    choose "Yes, they go to this school"
+    click_on "Continue"
+  end
+
+  def when_i_do_not_confirm_they_attend_the_pilot_school
+    expect(page).to have_content("Confirm your child’s school")
+    expect(page).to have_content("Pilot School")
+
+    choose "No, they go to a different school"
+    click_on "Continue"
+  end
+
+  def then_i_see_a_page_telling_me_i_cannot_continue
+    expect(page).to have_content(
+      "You cannot give or refuse consent through this service"
+    )
+  end
+
+  def and_i_fill_in_details_about_me
+    expect(page).to have_content("About you")
+    fill_in "Your name", with: "Jane #{@child.last_name}"
+
+    # Your relationship to the child
+    choose "Mum"
+
+    fill_in "Email address", with: "jane@example.com"
+    fill_in "Phone number", with: "07123456789"
+
+    click_on "Continue"
+  end
+
+  def and_i_provide_my_phone_contact_preferences
+    expect(page).to have_content("Phone contact method")
+    choose "I do not have specific needs"
+
+    click_on "Continue"
+  end
+
+  def and_i_give_consent_to_the_vaccination
+    expect(page).to have_content(
+      "Do you agree to them having the HPV vaccination?"
+    )
+    choose "Yes, I agree"
+    click_on "Continue"
+  end
+
+  def and_i_provide_my_childs_gp_details
+    expect(page).to have_content("Is your child registered with a GP?")
+    choose "Yes, they are registered with a GP"
+    fill_in "Name of GP surgery", with: "GP Surgery"
+    click_on "Continue"
+  end
+
+  def and_i_provide_the_home_address
+    expect(page).to have_content("Home address")
+    fill_in "Address line 1", with: "1 Test Street"
+    fill_in "Address line 2 (optional)", with: "2nd Floor"
+    fill_in "Town or city", with: "Testville"
+    fill_in "Postcode", with: "TE1 1ST"
+    click_on "Continue"
+  end
+
+  def and_i_answer_no_to_all_the_medical_questions
+    until page.has_content?("Check your answers and confirm")
+      choose "No"
+      click_on "Continue"
+    end
+  end
+
+  def then_i_see_a_confirmation_page
+    expect(page).to have_content("Check your answers and confirm")
+
+    expect(page).to have_content("Child’s name#{@child.full_name}")
+  end
+
+  def and_i_submit_the_consent_form
+    click_on "Confirm"
+    expect(page).to have_content(
+      "#{@child.full_name} will get their HPV vaccination at school"
+    )
+  end
+
+  def and_i_get_a_confirmation_email
+    # not sure how to do this yet
```

Comment:
A couple ways I can think of testing this.
- Use the rspec helper `have_enqueued_mail` to see if the correct `Mail` object is in the queue
- Use the rspec helper `have_been_delivered` to check if the correct `Mail` objects was delivered (e.g. was enqueued and then the queue was processed, I think?)
- Examine `ActionMailer::Base.deliveries` yourself, e.g. https://github.com/nhsuk/manage-childrens-vaccinations/blob/c2ea34b218196053277ee735fa8b0c9b6505c211/spec/jobs/consent_requests_session_batch_job_spec.rb. You can also examine the headers of each `Mail` object that was "delivered" if that's what you need.

Reply:
Thanks. I've given it a go here: 5cc0e7f3